protocols/identify/Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = "libp2p-identify"
 edition = "2021"
 rust-version = { workspace = true }
 description = "Nodes identifcation protocol for libp2p"
-version = "0.43.0"
+version = "0.43.1"
```

Comment:
Because `libp2p-swarm` is a dependency that is exposed in the public API, this is actually a breaking change for all protocol crates in one go.

For now, it is okay to leave this as is. We will make a PR that "open"s the merge window for breaking changes which bumps the versions accordingly.